fontFeatures/feaLib/Substitution.py

```diff
@@ -40,37 +40,86 @@ def paired_ligature(self):
             replacements.append(j)
     rhs = zip(*replacements)
     for l, r in zip(lhs,rhs):
-        b.statements.append(
-            feaast.LigatureSubstStatement(
+        stmt = feaast.LigatureSubstStatement(
             [glyphref(x) for x in self.precontext],
             [glyphref([x]) for x in l],
             [glyphref(x) for x in self.postcontext],
             glyphref([r[0]]),
             False,
         )
+        b.statements.append(stmt)
+    return b
+
+# Expand multiple substitutions, such that:
+#   * Substitute [a b] -> before_tail [a.2 b.2] tail;
+# Becomes in FEA:
+#   * sub a by before_tail a.2 tail;
+#   * sub b by before_tail b.2 tail;
+def paired_mult(self):
+    b = feaast.Block()
+
+    if len(self.input_lengths) != 1:
+        raise ValueError("Multiple substitution only valid on input of length one, use a Chain instead")
```

Comment:
One of my dreams for FEE is that it should be more of a compiler than a translator - i.e. instead of forcing the user to care about the OpenType "instructions" available, it will work out the best way to express the user's (high-level) intention in terms of those low-level instructions. But that is hard, and today is not the day for it. :-)